src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java

```diff
@@ -1,33 +1,65 @@
 package seedu.address.logic.commands;
 
+import static java.util.Objects.requireNonNull;
+
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.student.StudentNumber;
+import seedu.address.model.person.Name;
+import seedu.address.model.student.Student;
 
 /**
- * Deletes a student from the system.
+ * Deletes a student from the system based on the student number provided.
```

Comment:
This should be student name?

Reply:
Good catch, I've already modified it to delete based on student name. Thank you.